common/projectconfig/projectconfig.go

```diff
@@ -54,6 +55,26 @@ func GetDefaultConfigPath() string {
 	return filepath.Join(internal.GitRoot(""), "ftl-project.toml")
 }
 
+// CreateWritableFileIfNonexistent checks the last config file in the list of specified
+// paths and creates the file if it does not already exist.
+func CreateWritableFileIfNonexistent(ctx context.Context, paths []string) error {
+	logger := log.FromContext(ctx)
```

Comment:
You'll need to use `ConfigPaths()` to do the normalisation.

Reply:
done